packages/x-components/src/components/panels/base-tabs-panel.vue

```diff
@@ -287,7 +287,7 @@ behavior is disabled.
     activeTabClass="x-button-auxiliary"
     contentClass="x-padding--04 x-background--auxiliary"
     tabClass="x-button-ghost"
-    tabsListClass="x-list--horizontal"
+    tabListClass="x-flex-col"
```

Comment:
I guess this should be `x-flox-row` but it collides with `x-flex x-flex-col` added in line 5.
```suggestion
    tabListClass="x-flex-row"
```